aksel.nav.no/website/pages/templates/soknad-introside/intro.tsx

```diff
@@ -0,0 +1,359 @@
+import { useEffect } from "react";
+import { ArrowRightIcon } from "@navikt/aksel-icons";
+import {
+  Accordion,
+  Bleed,
+  BodyLong,
+  BodyShort,
+  Box,
+  Button,
+  Checkbox,
+  GuidePanel,
+  Heading,
+  Link,
+  List,
+  Page,
+  Show,
+  Stack,
+  VStack,
+} from "@navikt/ds-react";
+
+function Example() {
+  useDekorator();
+
+  return (
+    <Page data-aksel-template="form-intropage-v1" footer={<Footer />}>
+      <Header />
+      <VStack as="main" gap="8">
+        <Page.Block width="text" gutters>
+          <Bleed marginInline={{ lg: "24" }}>
+            <Stack
+              align="center"
```

Comment:
Is the pictogram supposed to be center aligned?
![image](https://github.com/navikt/aksel/assets/83693529/8e2934d5-61ab-407c-9e9e-894f52510225)

Reply:
It is how we agreed on it, because of some alignment issues when form id is not included.
However, I now fail to replicate the issue, so I can check again with Sjur.

Reply:
Seems like the issue no longer exists, so I top-aligned the pictogram again.